docs/requirements.txt

```diff
@@ -1,3 +1,3 @@
 myst-parser==0.17.*
 sphinx-rtd-theme==1.0.*
-sphinx==4.5.*
+sphinx<6
```

Comment:
To fix the pipeline error, I think you could set sphinx==5.3.*. The pipeline always takes version 4.5.* at this time.